kitsune/sumo/jinja2/base.html

```diff
@@ -75,6 +59,23 @@
   <![endif]-->
 
   {% include "includes/google-analytics.html" %}
+
+  {# TrafficCop experiment_fxa_cta_topbar #}
+  {% set run_experiments = False %}
+  {% if waffle.flag('experiment_fxa_cta_topbar') %}
+    {% if request.path in ['/en-US/kb/install-firefox-android-device-using-google-play',
```

Comment:
You should be able to do a single `if` here (with an `and`),  but if you think it's easier to read this way, I don't have a problem leaving as-is.